lib/grpc_or_grpcweb.dart

```diff
@@ -13,6 +13,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+export 'src/auth/auth.dart'
```

Comment:
One suggestion I could make here is to make a library `lib/src/shared/api.dart` or something like that and re-export it from here, `grpc.dart` and `grpc_web.dart` - this way when we update public interface we need to do it in one place instead of multiple places.